preconditions/iotics_test.sh

```diff
@@ -1,29 +1,25 @@
 #!/bin/bash
 
-IOTICSPACE_DOMAIN="https://ganymede.iotics.space"
-RESOLVER_DOMAIN="did.prd.iotics.com"
+SPACE_NAME="ganymede"
```

Comment:
https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/ 

`set -euo pipefail` below the shebang line.